vsintegration/src/FSharp.Editor/Options/EditorOptions.fs

```diff
@@ -58,7 +58,7 @@ type CodeFixesOptions =
       { // We have this off by default, disable until we work out how to make this low priority 
```

Comment:
Probably the comment is now wrong, right?

Reply:
No, this comment is referring to the Simplify Name analyzer, which is off by default due to performance reasons.

Reply:
Ahh ...